VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -46,7 +47,46 @@ def __init__(self,patching, hutil, logger,para_parser):
             errMsg = 'Failed to serialize customSettings with error: %s, stack trace: %s' % (str(e), traceback.format_exc())
             self.logger.log(errMsg, True, 'Error')
             self.isOnlyOSDiskBackupEnabled = False
-
+        # The command lsscsi is used for mapping the LUN numbers to the disk_names
+        self.command = "sudo lsscsi"
+        self.disksToBeIncluded = []
+        self.root_devices = []
+        self.root_mount_points = ['/' , '/boot/efi']
+        try:
+            self.lsscsi_list = (os.popen(self.command).read()).splitlines()
+        except Exception as e:
+            error_msg = "Failed to execute the command lsscsi because of error %s , stack trace: %s" % (str(e), traceback.format_exc())
+            self.logger.log(error_msg, True ,'Error')
+            self.output_lsscsi = ""
```

Comment:
this appears to be unused in other places. We can remove this if that is the case.